src/StandardError.js

```diff
@@ -85,6 +85,18 @@ const Internal = {
 		};
 	},
 
+	createError(error) {
+		return (details) => {
+
+			if (details == null) {
+				return { ...error };
+
```

Comment:
More extra whitespace I think

Reply:
I think for single line functions it's fine without the whitespace but I'm a big fan of adding space above an else block ot make the delineation more clear that there's another block in a clean way. Especially when you get into 'else if' territory, it helps keep it readable. No readability problems with a 1-line function though, so it's fine if it's a little more dense. Although, I 100% agree with the previous comment that it shouldn't be above the first if.